fastlane/lib/fastlane/runner.rb

```diff
@@ -149,6 +153,12 @@ def execute_action(method_sym, class_ref, arguments, custom_dir: '..')
       end
     end
 
+    def execute_flow_block(block, current_platform, lane, parameters)
+      # Call the platform specific block and then the general one
+      block[current_platform].call(lane, parameters) if block[current_platform] && current_platform
+      block[nil].call(lane, parameters) if block[nil]
```

Comment:
Okay, so I've tried a few ways:
```
      if block[current_platform] && current_platform
        block[current_platform].call(lane, parameters)
      elsif block[nil]
        block[nil].call(lane, parameters)
      end
```

and 

```
      block_to_call = block[current_platform] || block[nil]
      block_to_call.call(lane, parameters) if block_to_call
```

Neither of these actually work since it assumes we're calling one or the other. The real use case is that _both_ may be called, not one or the other.